### ISSUE TITLE
pylint/astroid TypeError C++ type 'QWidget*' is not supported as  type

### ISSUE THREAD

**Issue by thisch** [PyCQA/pylint#1783]
### Steps to reproduce
1.  run pylint in a codebase where pyqt is imported

### Current behavior

When 

```
pylint 1.6.5, 
astroid 1.4.5
Python 2.7.3 (default, Apr 14 2012, 08:58:41) [GCC]
```

is used, the following execption is shown in the output (in addition to the normal pylint output):
```
unexpected error while building astroid from living object
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/astroid/raw_building.py", line 316, in imported_member
    modname = getattr(member, '__module__', None)
TypeError: C++ type 'QWidget*' is not supported as  type
```

and 

```
No handlers could be found for logger "astroid.raw_building"
```
is shown in the output when

```
pylint 1.8.1, 
astroid 1.6.0
Python 2.7.3 (default, Apr 14 2012, 08:58:41) [GCC]
```
is used.

### Expected behavior

No spurious warning/exception is output.

**Comment by PCManticore** [comment]
I'm not sure this falls into pylint's scope. What you can try to do is to omit `PyQt` from being passed as a whitelisted extension, e.g `--extension-pkg-whitelist`, which means that it won't be imported any longer. Since we rely on this flag to pick what C extensions gets live analyzed, these could crash pylint as long as they have side efffects like this one
